xbmc/platform/win32/WIN32Util.cpp

```diff
@@ -1196,7 +1196,38 @@ HDR_STATUS CWIN32Util::ToggleWindowsHDR(DXGI_MODE_DESC& modeDesc)
   HDR_STATUS status = HDR_STATUS::HDR_TOGGLE_FAILED;
 
 #ifdef TARGET_WINDOWS_STORE
-  // Not supported - not implemented yet
+  auto hdmi = HdmiDisplayInformation::GetForCurrentView();
+
+  if (!hdmi)
+    return status;
+
+  const auto current = hdmi.GetCurrentDisplayMode();
+
+  const bool needHDR = (current.ColorSpace() != HdmiDisplayColorSpace::BT2020);
+
+  for (const auto& mode : hdmi.GetSupportedDisplayModes())
+  {
+    if ((!needHDR || (needHDR && mode.ColorSpace() == HdmiDisplayColorSpace::BT2020)) &&
```

Comment:
Not sure mode.ColorSpace() is the best check, the values don't match the values of HdmiDisplayHdrOption passed to RequestSetCurrentDisplayModeAsync

mode also has attributes "IsSdrLuminanceSupported" "IsSmpte2084Supported" "IsDolbyVisionLowLatencySupported", which match the values of HdmiDisplayHdrOption exactly.
But MS doc is not really clear...

Reply:
Changed to use `IsSmpte2084Supported()` in toggle code